aptible/environment.go

```diff
@@ -0,0 +1,90 @@
+package aptible
+
+import (
+	"fmt"
```

Comment:
The linter is complaining about this for some reason:

```
aptible/environment.go:4: File is not `goimports`-ed (goimports)
	"fmt"
```

Reply:
Maybe it has to do with grouping?

Reply:
might be, updated and seeing if it works